tfx/components/example_validator/executor_test.py

```diff
@@ -29,45 +29,72 @@
 
 class ExecutorTest(tf.test.TestCase):
 
-  def testDo(self):
-    source_data_dir = os.path.join(
-        os.path.dirname(os.path.dirname(__file__)), 'testdata')
+  def setUp(self):
+    super(ExecutorTest, self).setUp()
+    self.source_data_dir = os.path.join(
+        "../", os.path.dirname(os.path.dirname(__file__)), 'testdata')
```

Comment:
I don't think you want the prepended "../".

Reply:
You're right, I removed the "../"